proto/mls/message_contents/commit_log.proto

```diff
@@ -0,0 +1,59 @@
+// Defines entries on the commit log, used for fork detection and recovery
+syntax = "proto3";
+
+package xmtp.mls.message_contents;
+
+enum CommitResult {
+  COMMIT_RESULT_UNSPECIFIED = 0;
+  COMMIT_RESULT_SUCCESS = 1;
+  COMMIT_RESULT_WRONG_EPOCH = 2;
+  COMMIT_RESULT_UNDECRYPTABLE = 3;
+  COMMIT_RESULT_INVALID = 4;
+}
+
+/**
```

Comment:
I'll remove all these comments before landing. Just wrote them here before I add them to the XIP